test/newNameLength.test.ts

```diff
@@ -2,6 +2,7 @@ import 'source-map-support/register';
 import * as cdk from 'aws-cdk-lib';
```

Comment:
Let's rename this file to `contraints-validation.test.ts`

Reply:
Looks like the name stayed the same.

Reply:
I just changed it this morning and have not committed yet